### DISCUSSION TITLE
Disable Nova Notifications (v4)

### DISCUSSION THREAD

**Discussion by clementmas** [laravel/nova-issues#4004] [Ideas & Feature Requests]
Is there a way to disable the new notification system in Nova 4?
That means hiding the icon, the panel and stopping the polling feature which makes unnecessary HTTP requests.

It's a nice addition but I don't think it should be "mandatory" to use it.

Thanks!

**Comment by jjjrmy** [comment]
This just stops it from polling constantly, you can use this until they allow you to remove Notifications totally.

`NovaServiceProvider`
```php
Nova::notificationPollingInterval(86400);
```

**Reply by clementmas** [reply]
Good point. I'll use that meanwhile. Thanks!

**Comment by crynobone** [comment]
https://nova.laravel.com/docs/4.0/customization/notifications.html#disabling-notifications

**Reply by trovster** [reply]
This doesn't appear to work. I have added `Nova::withoutNotificationCenter();` to the Nova Provider, but the icon still exists and the requests are still being make. Similarly `withoutThemeSwitcher()` doesn't remove the theme switcher icon which it should do according to the code.

**Reply by davidhemphill** [reply]
Make sure you've updated your assets by running `php artisan nova:publish --force` after `composer update`.

**Reply by trovster** [reply]
Perfect :)

**Comment by RhysLees** [comment]
It would be nice if we could use WebSockets/echo for notifications optionally of course#4196
